src/routes/docs/tooling/command-line/installation/+page.markdoc

```diff
@@ -156,6 +228,12 @@ $APPWRITE_INSTALL_DIR = Join-Path -Path $env:LOCALAPPDATA -ChildPath "Appwrite";
 rm -f /usr/local/bin/appwrite | bash
 ```
 {% /tabsitem %}
+
+{% tabsitem #scoop title="Scoop" %}
+```sh
+scoop uninstall ./appwrite
```

Comment:
Does using a relative path really work?

Reply:
While it does work, I did confirm using the command `scoop uninstall appwrite` also works, so I will be updating this uninstall scoop command instead of using a relative path here.